Tasks/UsePythonVersionV0/task.json

```diff
@@ -43,6 +43,13 @@
             "defaultValue": "false",
             "helpMarkDown": "Allow downloading unstable python versions from [Github Actions python versions registry](https://github.com/actions/python-versions)."
         },
+        {
+            "name": "githubToken",
+            "type": "string",
+            "label": "GitHub token",
+            "required": false,
+            "helpMarkDown": "GitHub token that will be used for downloading python versions from [Github Actions python versions registry](https://github.com/actions/python-versions). Not needed if you don't need to download python."
```

Comment:
Could it make sense to describe the purpose here? Generally download should work without it, but it will allow to avoid anonymous requests limit in GitHub

Reply:
Clarified the description